app/views/dashboard/districts/preview.html.erb

```diff
@@ -1,270 +1,9 @@
-            <path d="M2 3h6a4 4 0 0 1 4 4v14a3 3 0 0 0-3-3H2z"></path>
-            <path d="M22 3h-6a4 4 0 0 0-4 4v14a3 3 0 0 1 3-3h7z"></path>
-          </svg>
-        </div>
-        <p class="font-sans font-semibold text-md text-grey-500">
-          Resources
-        </p>
-      </div>
-    </a>
-  </div>
-  <div class="mb-1 px-2">
-    <a class="block" href="https://www.simple.org/privacy">
-      <div class="flex py-2 px-3 bg-white rounded-md transition-bg duration-250 ease-in-expo">
-        <div class="relative top-1 mr-2">
-          <svg class="stroke-current text-grey-300" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round">
-            <path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z"></path>
-          </svg>
-        </div>
-        <p class="font-sans font-semibold text-md text-grey-500">
-          Privacy Policy
-        </p>
-      </div>
-    </a>
-  </div>
-  <div class="mb-3 px-2">
-    <a class="block" href="https://www.simple.org/privacy">
-      <div class="flex py-2 px-3 bg-white rounded-md transition-bg duration-250 ease-in-expo">
-        <div class="relative top-1 mr-2">
-          <svg class="stroke-current text-grey-300" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round">
-            <path d="M14 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V8z"></path>
-            <polyline points="14 2 14 8 20 8"></polyline>
-            <line x1="16" y1="13" x2="8" y2="13"></line>
-            <line x1="16" y1="17" x2="8" y2="17"></line>
-            <polyline points="10 9 9 9 8 9"></polyline>
-          </svg>
-        </div>
-        <p class="font-sans font-semibold text-md text-grey-500">
-          License
-        </p>
-      </div>
-    </a>
-  </div>
-  <div class="mb-1 pt-4 px-6 border-t border-grey-200">
-    <div class="mb-3">
-      <p class="font-sans font-semibold text-md text-grey-500">
-        Account
-      </p>
-      <p class="font-sans font-medium text-sm text-grey-400">
-        Signed in as
-        <span class="font-semibold">
-          cvho@simple.org
-        </span>
-      </p>
-    </div>
-    <a class="flex align-center justify-center py-2 px-4 bg-white rounded-md border border-grey-200" href="/" type="button">
-      <div class="flex">
-        <div class="relative top-1 mr-2 opacity-50">
-          <svg class="stroke-current text-red-200" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round">
-            <path d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"></path>
-            <polyline points="16 17 21 12 16 7"></polyline>
-            <line x1="21" y1="12" x2="9" y2="12"></line>
-          </svg>
-        </div>
-        <p class="font-sans text-md font-semibold text-red-200">
-          Logout
-        </p>
-      </div>
-    </a>
-  </div>
-</div>
```

Comment:
@claudiovallejo just a heads up, I extracted the nav bar to a partial so I can use it from the new layout.